### PR TITLE
Align neural speed python api

### PR DESCRIPTION
**Pull request by intellinjun**
## Type of Change

feature or bug fix or documentation or others
API changed or not

## Description

detail description 
Issues: xxx

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR 

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

any library dependency introduced or removed

### CODE REVIEW

**Review by zhenwei-intel — APPROVED** [review]
LGTM

File: neural_speed/__init__.py [hunk 1]
```diff
@@ -73,67 +73,70 @@ def get_model_type(model_config):
             model_type = "chatglm2"
         return model_type
 
-    def init(self, model_name, not_quant=False, use_cache=False, use_gptq=False, use_awq=False,
-            weight_dtype="int4", alg="sym", group_size=32,
-            scale_dtype="fp32", compute_dtype="int8", use_ggml=False):
+    def init(self, model_name, use_quant=True, use_gptq=False, **quant_kwargs):
```
Comment (a32543254) [inline neural_speed/__init__.py:76]: why remove the default value of init args ?

File: neural_speed/__init__.py [hunk 2]
```diff
@@ -73,67 +73,70 @@ def get_model_type(model_config):
...
             self.bin_file = quant_bin
-        if use_cache and os.path.exists(self.bin_file):
+
+        if os.path.exists(self.bin_file):
+            print("{} existed, will use cache file. Otherwise please remove the file".
+                  format(self.bin_file))
             return
 
-        if use_gptq or use_awq:
+        if use_gptq:
```
Comment (a32543254) [inline neural_speed/__init__.py:109]: keep awq

File: neural_speed/__init__.py (hunk 1 above)
Comment (intellinjun) [reply neural_speed/__init__.py:76]: There was no default value on ITREX back then.

Comment (intellinjun) [reply neural_speed/__init__.py:76]: I will ask zhenwei how to do it.

Comment (intellinjun) [reply neural_speed/__init__.py]: done

File: neural_speed/__init__.py (hunk 2 above)
Comment (intellinjun) [reply neural_speed/__init__.py]: done

**Review by DDEle — APPROVED** [review]
LGTM

**Pull request merged**